app/request-actions/private-cloud/edit-request.ts

```diff
@@ -75,7 +88,7 @@ export default async function editRequest(
 
   const { changes, ...otherChangeMeta } = comparePrivateProductData(rest, previousRequest?.decisionData);
 
-  const quotaChangeInfo = isNoQuotaChanged
+  const quotaChangeInfo = noQuotaChange
```

Comment:
const quotaReviewResult = await checkIfQuotaAutoApproval(currentQuota, requestedQuota, project.licencePlate, project.cluster);

const quotaChangeInfo = quotaReviewResult.hasQuotaChanged > then include the additional fields.